posts/2023-12-21_end_of__year__up.../end_of__year__update_from_the__pharmaverse__council.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "End of Year Update from the Pharmaverse Council"
+author:
+  - name: Mike Stackhouse
+description: ""
+date: "2023-12-21"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [community]
+# feel free to change the image
+image: "pharmaverse.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2023-12-21_end_of__year__up..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+## 2023 Was a Big Year
+
+This was a big year for open-source work in clinical submissions in general. We saw Roche speak about shifting to an [open-source backbone for clinical trials](https://www.youtube.com/watch?v=nqJsLSLd39A). Novo Nordisk spoke publicly of an [R based submission to the FDA](https://www.youtube.com/watch?v=t33dS17QHuA). These are true marks of progress being made in R becoming a first class language for clinical reporting. 
+
+Back in August, Nicholas Eugenio released a [blog post](https://pharmaverse.github.io/blog/posts/2023-10-10_pharmaverse_story/pharmaverse_story.html) on this history of pharmaverse. It's funny to think about the fact that only 3 years ago, the idea of cross organization collaboration on R packages and building a community around this in the clinical world was just a conversation between friends. Since then, we have a community of more than 1200 people on Slack, 350 on LinkedIn, interest from over 150 organizations, and over 30 packages. If 2020 through 2022 was the birth of Pharmaverse, 2023 was finding our identity as a community. For the council, 2024 will be about continuing to mature and find more ways that we can continue to support the community.
+
+## What We Accomplished
+
+One of our biggest moves in 2023 was to form our partnership with [PHUSE](https://phuse.global/PHUSE_Open_Source_Technology). This latched us into an existing community with shared values and a platform that helps us continue to build the pharmaverse community. At the [PHUSE EU Connect](https://www.phuse-events.org/attend/frontend/reg/thome.csp?pageID=11403&eventID=19&traceRedir=2) we had our first opportunity to host a pharmaverse meetup and bring together pharmaverse contributors in person. As we move forward, we'll continue to use this platform to find ways we can host events and encourage collaboration within the pharmaverse community.
```

Comment:
Maybe add a short mention for the PHUSE EU Connect panel session? I felt that was a nice highlight of the year to bring together so many diverse OS experts.

Reply:
Agree, I would also add that if we have any pictures form that it would be nice to include the best one below!

Reply:
I don't this we have any - @rossfarrugia  do you?

Reply:
@MichaelRimler i think Kayley took one right of the panelists? i see it on this blog https://phuse.global/Communications/PHUSE_Blog/eu-connect-2023-summary-and-award-winners